### ISSUE TITLE
"Bad muon" filters for re-miniAOD

### ISSUE THREAD

**Issue by andresti** [vhbb/cmssw#670]
In case of running on re-miniAOD, we need to save the following flags to ntuples for "bad muon" events (as described in https://twiki.cern.ch/twiki/bin/view/CMS/MissingETOptionalFiltersRun2#Moriond_2017):

    Bad PF Muon Filter
    Bad Charged Hadrons
    Flag_badMuons -> New Giovanni's Filter that the MET is corrected for (flag is set to true if the MET got corrected)
    Flag_duplicateMuons -> New Giovanni's Filter that the MET is corrected for (flag is set to true if the MET got corrected) 

Or is that covered by https://github.com/vhbb/cmssw/issues/603 ?

**Comment by jpata** [comment]
@andresti do you already have some code for that? I think this should mostly be accessible via the HLT bits.

**Comment by jpata** [comment]
Looking deeper it needs some additional CMSSW modules:

https://gitlab.cern.ch/ttH/reference/blob/master/definitions/Moriond17.md#42-met-filters

**Comment by jpata** [comment]
IN particular, we'd need to merge https://github.com/vhbb/cmssw/compare/vhbbHeppy80X...cms-met:METRecipe_80X_part2 but I think this requires a CMSSW bump as it's conflicting.

**Comment by andresti** [comment]
@jpata no, I don't have any code for that, I was wondering if there already was something.
